src/test/samples/javascript/sampleFunctions.js

```diff
@@ -6,26 +6,29 @@
 'use strict'
 
 function functionWithNoArgs() { }
-
-export function exportedFunctionWithNoArgs() { }
-exports.exportedFunctionWithNoArgs = exportedFunctionWithNoArgs
+exports.exportedFunctionWithNoArgs = functionWithNoArgs
```

Comment:
Are there tests elsewhere that over this? `export function // ...` is still a valid case that we need to test.

Reply:
For now this syntax is exercized in sampleFunctions.ts